recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,4 @@
+c_compiler:  # [win]
+  - vs2019  # [win]
```

Comment:
Can you add a note that says "Upstream pandas says that 2022 because they use C11. But we found that we can still use 2029 if we use the Windows 10 SDK 19041, which is available on our builder machines".

I feel like this is important considering the amount of time we spent on fixing this :)